### PR TITLE
Changes on MAI and AVAX on Avalanche

### PR DESCRIPTION
**Pull request by eboadom**
MAI
- Enabled as collateral in isolation. 2m debt ceiling.
- 50m supply cap.
- Not borrowable in isolation.

FRAX
- Not borrowable in isolation

### CODE REVIEW

File: script/DeployAvaFRAXSteward.s.sol [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import 'forge-std/console.sol';
+import {Script} from 'forge-std/Script.sol';
+import {AaveV3AvaFRAXListingSteward} from '../src/contracts/frax/AaveV3AvaFRAXListingSteward.sol';
+
+contract DeployAvaFRAXSteward is Script {
+    function run() external {
+        vm.startBroadcast();
+        AaveV3AvaFRAXListingSteward fraxSteward = new AaveV3AvaFRAXListingSteward();
+        console.log('Frax Steward address', address(fraxSteward));
```
Comment (sakulstra) [inline script/DeployAvaFRAXSteward.s.sol:12]: is this needed? i think broadcast will automatically log all deployed contract addresses

File: src/test/MAIAaveV3AvaListingByGuardian.t.sol [hunk 2]
```diff
@@ -22,6 +22,10 @@ contract MAIAaveV3AvaListingByGuardian is Test {
     address public constant MAI_WHALE =
         0xbE56bFF41AD57971DEDfBa69f88b1d085E349d47;
 
+    // It is not so simple to get the biggest holders of MAI on Avalanche, so we will
+    // send funds on setup
+    address public constant MAI_WHALE_2 = address(1);
```
Comment (sakulstra) [inline src/test/MAIAaveV3AvaListingByGuardian.t.sol:27]: why doesn't deal work?

**Review by sakulstra — APPROVED** [review]

File: src/test/MAIAaveV3AvaListingByGuardian.t.sol (hunk 2 above)
Comment (eboadom) [reply src/test/MAIAaveV3AvaListingByGuardian.t.sol:27]: Didn't try actually. Can do

File: script/DeployAvaFRAXSteward.s.sol (hunk 1 above)
Comment (eboadom) [reply script/DeployAvaFRAXSteward.s.sol:12]: True, I was not aware of that before

**Pull request merged**